README.md

```diff
@@ -1,2 +1,59 @@
-# snowplow-aws-lambda-source
+# Snowplow AWS Lambda source
 
+[ ![Build Status] [travis-image] ] [travis]
+[ ![Release] [release-image] ] [releases]
+[ ![License] [license-image] ] [license]
+
+Snowplow AWS Lambda source is a tool to track [AWS Lamba event sources](http://docs.aws.amazon.com/lambda/latest/dg/intro-core-components.html#intro-core-components-event-sources) in [Snowplow](http://snowplowanalytics.com/).
+
+## Currently supported sources
```

Comment:
README.md should be much simplified, like a tracker README. Should contain a table with 4 links, per the tracker READMEs.